front-end/src/components/MainPage.jsx

```diff
@@ -19,6 +19,42 @@ class MainPage extends Component {
     store.dispatch(openAnalysisPage());
   }
 
+  downloadTranscriptTemplate = () => {
+    const element = document.createElement("a");
+    const data = {"data": [
+        {
+          "": "23606",
+          "project_id": "1",
```

Comment:
Maybe make it more clear that we would like them to change their project ID. i.e. 
```suggestion
          "project_id": "<your unique project ID>",
```